mu/locale/pt_BR/LC_MESSAGES/mu.po

```diff
@@ -118,42 +126,48 @@ msgid ""
 " Connect your device<br/>&nbsp;4. Load the .bin file below using the "
 "'Browse' button<br/>&nbsp;5. Press 'Erase & write firmware'"
 msgstr ""
+"&nbsp;1. Determine o tipo de dispositivo (ESP8266 ou ESP32)<br />&nbsp;2."
+" Baixe o firmware do <a href=\"https://micropython.org/download\" "
+"style=\"color:#039be5;\">https://micropython.org/download</a><br/>&nbsp;3."
+" Conecte seu dispositivo<br/>&nbsp;4. Carregue o arquivo .bin abaixo "
+"usando o botão \"Procurar\"<br/>&nbsp;5. Pressione \"Apagar e escrever "
+"firmware\""
 
 #: mu/interface/dialogs.py:334 mu/modes/web.py:82
 msgid "Browse"
-msgstr ""
+msgstr "Procurar"
```

Comment:
For  `mu/modes/web.py:82`, this is meant as "browse a website", which would probably be "Navegar". For `mu/interface/dialogs.py:334` it's "browse for a file" IIUC, and "Procurar" fits well there. Maybe we should have two separate entries for these?

Reply:
Totally agree. Setting msgctxt for each case would be perfect for splitting "Browse" in the PO file.